scrapper/controllers/__init__.py

```diff
@@ -4,21 +4,17 @@
 from flask import render_template, request, redirect, Blueprint, abort
 from flask.ext.api import status
 
+from scrapper.controllers.twitter import twitter_bp
 from scrapper.controllers.miner import mine_user
 
 
-twitter = Blueprint('twitter', __name__, url_prefix = '/twitter')
-
-@twitter.route('/', methods=['GET', 'POST'])
+@twitter_bp.route('/', methods=['GET', 'POST'])
 def index():
     abort(400)
 
-@twitter.route('/<user_name>')
+@twitter_bp.route('/<user_name>')
 def show_user(user_name=None):
     refresh = request.args.get('refresh', '')
-    user = miner.mine_user(user_name, refresh)
-    user = json.dumps(user)
+    user = miner.mine_user.delay(user_name, refresh)
```

Comment:
Also, keep in mind tasks are asynchronous. This might be executed only 5 minutes from the request time. Doing it like this will keep the connection waiting.